includes/class-wc-admin-reports-customers-query.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/**
+ * Class for parameter-based Customers Report querying
+ *
+ * Example usage:
+ * $args = array(
+ *          'registered_before'   => '2018-07-19 00:00:00',
+ *          'registered_after'    => '2018-07-05 00:00:00',
+ *          'page'                => 2,
+ *          'avg_order_value_min' => 100,
+ *          'country'             => 'GB',
+ *         );
+ * $report = new WC_Admin_Reports_Customers_Query( $args );
+ * $mydata = $report->get_data();
+ *
+ * @package  WooCommerce Admin/Classes
+ */
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Reports_Customers_Query
+ */
+class WC_Admin_Reports_Customers_Query extends WC_Admin_Reports_Query {
+
+	/**
+	 * Valid fields for Customers report.
+	 *
+	 * @return array
+	 */
+	protected function get_default_query_vars() {
+		return array(
+			'per_page' => get_option( 'posts_per_page' ), // not sure if this should be the default.
```

Comment:
Seems okay to me - I think be default we are doing, seems like most of the query classes omit this default query.